crates/biome_js_analyze/src/lint/nursery/use_semantic_elements.rs

```diff
@@ -59,25 +63,83 @@ impl Rule for UseSemanticElements {
             let is_not_interative =
                 AriaRoles.is_not_interactive_element(element_name, extract_attributes);
```

Comment:
I forgot to mention this before, but don't import `AriaRoles` like this. Use `let aria_roles = ctx.aria_roles();` instead.